packages/api/src/routes.js

```diff
@@ -30,6 +30,11 @@ module.exports = ({
       method: 'GET',
       handler: rateController.getUSDRate
     },
+    {
```

Comment:
Why this is needed?

Reply:
No params, maybe hardcode the controller invokation then?

Reply:
it's necessary to get the last epoch